apps/rush-lib/src/logic/base/BaseInstallManager.ts

```diff
@@ -259,13 +242,17 @@ export abstract class BaseInstallManager {
       console.log('');
     }
 
-    if (!this.options.noLink) {
-      const linkManager: BaseLinkManager = LinkManagerFactory.getLinkManager(this._rushConfiguration);
-      await linkManager.createSymlinksForProjects(false);
-    } else {
-      console.log(
-        os.EOL + colors.yellow('Since "--no-link" was specified, you will need to run "rush link" manually.')
-      );
+    // Skip linking if using workspaces
+    if (!useWorkspaces) {
+      if (!this.options.noLink) {
```

Comment:
Do we show an error if `--no-link` is used in a workspaces repo?

Reply:
Yes we do :)